creator/ingest_runs/data_generator/ingest_package/extract_configs/biospec.py

```diff
@@ -22,14 +22,20 @@
 
 # TODO - Replace this with operations that make sense for your own data file
 operations = [
-    value_map(
```

Comment:
We shouldn't mutate the original id so just use keep map for all the external ids coming from our generated study